### ISSUE TITLE
Typescript types are missing playPosition

### ISSUE THREAD

**Issue by sampsonjoliver** [react-native-google-cast/react-native-google-cast#155]
Simple enough. In `3.2.1` the `CastOptions` are missing `playBackPosition`

```
  export type CastOptions = {
    title?: string
    subtitle?: string
    studio?: string
    imageUrl?: string
    posterUrl?: string
    isLive?: boolean
    mediaUrl: string
    contentType?: string
    customData?: any
    streamDuration?: number
    textTrackStyle?: TextTrackStyle

  // missing
  playPosition?: number
  }
```

As we rely on this for our implementation, this blocks our build due to typescript errors.

**Comment by petrbela** [comment]
Fixed in 3.2.2. Thanks!

**Comment by sampsonjoliver** [comment]
Awesome, thank you! :)
